controllers/kubeadmconfig_controller.go

```diff
@@ -95,6 +96,8 @@ func (r *KubeadmConfigReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, re
 	machine, err := util.GetOwnerMachine(ctx, r.Client, config.ObjectMeta)
 	if err != nil {
 		log.Error(err, "could not get owner machine")
+                config.Status.ErrorStatus = CreateKubeadmconfigError
```

Comment:
Should this be a non-retryable error? I would probably expect this to be a transient error only.

We should only set ErrorMessage and ErrorReason for non-retryable errors.

Reply:
Agreed, I missed this change when I first glanced through the PR.

@mytunguyen ErrorStatus and ErrorReason are usually only set in failures, when you usually require user intervention for example.

Reply:
makes sense. okay!